localstack/utils/analytics/metadata.py

```diff
@@ -94,6 +105,26 @@ def get_machine_id() -> str:
     return doc["machine_id"]
 
 
+def get_localstack_edition() -> Optional[Literal["enterprise", "pro", "community"]]:
+    if os.path.exists("/usr/lib/localstack/.enterprise-version"):
+        return "enterprise"
+    elif os.path.exists("/usr/lib/localstack/.pro-version"):
+        return "pro"
+    elif os.path.exists("/usr/lib/localstack/.community-version"):
+        return "community"
+
+    return None
+
+
+def is_license_activated() -> bool:
+    try:
```

Comment:
nit: Maybe it would make sense to separate the imports here:
- First try to import anything super-stable from ext.
  - return false if it's not possible
- Then try to import the specific method.
  - list a prominent error in this case, because that would mean that this code stopped working because of a refactoring, not because the package is missing